backend/src/main/resources/static/auth.yml

```diff
@@ -0,0 +1,117 @@
+openapi: "3.0.2"
+info:
+  title: Authentication API
+  description: API for user registration and login
+  version: "1.0"
+servers:
+  - url: http://localhost:8090/api
+    description: Local development server
+tags:
+  - name: Auth
+    description: Operations for user authentication
+
+paths:
+  /auth/register:
+    post:
+      tags:
```

Comment:
try to register "operationName" in order to take control over the generated java methods